gems/smithy-client/lib/smithy-client/codecs/cbor.rb

```diff
@@ -30,7 +30,8 @@ def serialize(data, shape)
         # @param [Struct] type
         # @return [Object, Hash]
         def deserialize(bytes, shape, type = nil)
-          return {} if bytes.empty?
+          return {} # for development purposes since default response body is StringIO
+          # return {} if bytes.empty?
```

Comment:
I noticed that in the smithy-decaf implementation for response class - we did not end up implementing `body_contents` method ([source](https://github.com/aws/aws-sdk-ruby/blob/version-3/gems/aws-sdk-core/lib/seahorse/client/http/response.rb#L45-L56)) - I was wondering if we should migrate that functionality over to cover eventstreams and converting io -> string. Thoughts?